### PR TITLE
Added errorformat for stylelint

### PR DESCRIPTION
**Pull request by rhysd**
[stylelint](https://github.com/stylelint/stylelint) is a de facto linter for CSS.

### CODE REVIEW

**Comment by haya14busa** [comment]
Thank you for the Pull Request! :+1: :dog:

image

It seems that the result may contain danger mark, maybe as a warning.
Could you please considering to add support this as well?

**Comment by rhysd** [comment]
Ah, I forgot it.  I'll fix it tonight (sorry I can't have a time to fix for now)

**Comment by haya14busa** [comment]
No problem!

File: fmts/doc.go [hunk 1]
```diff
@@ -1,7 +1,9 @@
 // Package fmts holds defined errorformats.
 //
 // Defined formats:
-// 
+//
+// 	css
+// 		stylelint	A mighty modern CSS linter - https://github.com/stylelint/stylelint
```
Comment (haya14busa) [inline fmts/doc.go]: Could you use `go generate ./...` to generate document. 
https://github.com/haya14busa/errorformat/tree/master/fmts#4-go-generate-

Comment (rhysd) [inline fmts/doc.go]: I didn't understand that. I'll do :dog:

**Comment by rhysd** [comment]
Problems should have gone!

File: fmts/css.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package fmts
+
+func init() {
+	const lang = "css"
+
+	register(&Fmt{
+		Name: "stylelint",
+		Errorformat: []string{
+			`%+P%f`,
```
Comment (haya14busa) [inline fmts/css.go]: I'm sorry for additional comment but I think `%-P%f` is better.
By using `-` flag, first `renderer/style.css|| renderer/style.cs` line will be removed.

Comment (rhysd) [inline fmts/css.go]: :+1: It must be beter

**Comment by rhysd** [comment]
I fixed the point.

**Pull request merged**